scripts/update-tezos.sh

```diff
@@ -8,14 +8,14 @@
 
 set -e
 
-git config user.name "serokell-bot" # necessary for pushing
-git config user.email "tezos-packaging@serokell.io"
+git config user.name "Savely Krendelhoff" # necessary for pushing
+git config user.email "alicecandyom@proton.me"
```

Comment:
This seems like a leftover change that should be removed

Same for the other changes in this file and commit (dd0887e1ba2db35ce28ad04515fdc2d1167f0824)